10 Ditto/10Ditto.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+  </head>
+  <body>
+
+    <script>
+    let obj1 = {thing : "hi",
```

Comment:
These can all be `const` since we aren't reassigning* it to anything else.

* Reassigning means setting the variable with another value. So in this case, we are not doing `obj1 = 1` and `obj1 = 2`. Notice how we only write `obj =` once.